hack/scale_out_poc/nginx.conf

```diff
@@ -71,11 +71,11 @@ http {
                     proxy_pass {{ arktos_api_protocol }}://$remote_addr:{{ arktos_api_port }};
                 }
 
-                location ~ ^/api/v1/nodes?(.*) {
+                location ~ ^/api/([^/])*/nodes?(.*) {
                     proxy_pass $resource_api;
                 }
 
-                location ~ ^/apis/coordination.k8s.io/v1/leases?(.*) {
```

Comment:
there used to be code to fall back to remote_addr:8080 whenever the location is not found, which I have in my tests env with the PR 810 code. any reason those default location was removed ? the fallback to default should address such issues --- which was the reason I didn't see this in my env.

Reply:
This is for m1. The one you mentioned is a different file, which is for m2 only. 

If we use the logic to remote_addr:8080 whenever the location is not found in m1, the requests for pods/daemonsets from the components in RP will not be forwarded to TP side.